website/docs/docs/build/custom-schemas.md

```diff
@@ -83,30 +84,23 @@ The following code represents the default macro's logic:
 {%- endmacro %}
 ```
 
-## Advanced custom schema configuration
-
-You can customize schema name generation in dbt depending on your needs, such as creating a custom macro named `generate_schema_name` in your project or using the built-in macro for environment-based schema names. The built-in macro follows a pattern of generating schema names based on the environment, making it a convenient alternative.
-
-If your dbt project has a macro that’s also named `generate_schema_name`, dbt will always use the macro in your dbt project instead of the default macro.
-
-### Changing the way dbt generates a schema name
+## Changing the way dbt generates a schema name
 
-To modify how dbt generates schema names, you should add a macro named `generate_schema_name` to your project and customize it according to your needs:
+If your dbt project has a custom macro called `generate_schema_name`, dbt will use it instead of the default macro. This allows you to customize the name generation according to your needs.
 
-- Copy and paste the `generate_schema_name` macro into a file named 'generate_schema_name'.
+To customize this macro, copy the example code above into a file named `macros/generate_schema_name.sql` and make changes as necessary.
```

Comment:
sugg: we've started to avoid using words like "below" and "above" to use more inclusive language for less-abled people

```suggestion
To customize this macro, copy the example code into a file named `macros/generate_schema_name.sql` and make changes as necessary.
```